cairo/ethereum/cancun/vm/interpreter.cairo

```diff
@@ -308,6 +309,28 @@ func execute_code{
 
     // TODO: Handle precompiled contracts
 
+    // code_address might be optional in create scenarios at this point.
+    if (cast(evm.value.message.value.code_address.value, felt) != 0) {
+        let (precompile_address, precompile_fn) = precompile_table_lookup(
+            [evm.value.message.value.code_address.value]
+        );
+        // Addresses that are not precompiles return 0.
+        if (precompile_address != 0) {
```

Comment:
Note that errors have to be handled at some point 
https://github.com/ethereum/execution-specs/blob/d9febcb7332d62194968dcdb7775adf497c93626/src/ethereum/cancun/vm/interpreter.py#L288